src/utils.cpp

```diff
@@ -1610,6 +1610,174 @@ bool biocyber_compatibility(struct obj_data *obj1, struct obj_data *obj2, struct
           return FALSE;
         }
         break;
+      case CYB_CYBERARMS:
```

Comment:
This won't work as written. Think of a switch statement like a chain of if-elseif-else. What you have is:

```
if the type is cyberarms, then do this
otherwise, if the type is cyberarms, then do this
otherwise...```

Instead, remove all the extraneous CYB_CYBERARMS lines and breaks, and just stick all your if statements together.

Reply:
Hopefully this is what you meant

Reply:
Yep! However, it won't compile since there are a bunch of opening braces without closing braces. You gotta either take off the ending braces of `if (x) {` or add closing braces to the lines that use them, like `} if (x) {`. I personally prefer the latter style, but either way is fine.